### ISSUE TITLE
Regarding the mIoU of the KITTI dataset

### ISSUE THREAD

**Issue by Songhaijian-H** [valeoai/WaffleIron#19]
Hello, first of all, thank you for providing such a great paper. My question is, which config file did you use when training on the KITTI dataset? I was unable to reproduce the results, as my mIoU did not reach as high as 70.8.Additionally, I couldn't find the PolarMix code in the repository. Could you please tell me where it is located?Thank you.

**Comment by gpuy** [comment]
Hello,

The config file for KITTI is located [here](https://github.com/valeoai/WaffleIron/blob/master/configs/WaffleIron-48-256__kitti.yaml).

Polarmix and cutmix are activated by default in this config file (see [here](https://github.com/valeoai/WaffleIron/blob/master/configs/WaffleIron-48-256__kitti.yaml#L45)). These augmentations are called [here](https://github.com/valeoai/WaffleIron/blob/master/datasets/semantic_kitti.py#L297). The implementation of both augmentations is a bit higher up in the same file (only implemented for SemanticKITTI).

If you launch the training on the train set and evaluate on the validation set, you should reach an mIoU of around 68.0.

The mIoU of 70.8 is obtained on the test set. We used the config file above with recipes used in previous works when submitting on the test set:
- train on the combination of the training and validation set using the argument `--trainval` when launching training. See explanation [here](https://github.com/valeoai/WaffleIron/blob/master/README.md#semantickitti-1).
- [inference](https://github.com/valeoai/WaffleIron/blob/master/README.md#semantickitti) is done with test time augmentation. You need to add the option `--num_votes 12` when calling this script and using `--batch_size 12` (you can also use any batch size that divide 12).

Did you follow these steps?

**Comment by Songhaijian-H** [comment]
Thank you very much for your prompt response. After reviewing your answer, I realize that I did not follow the parameters you suggested for training. Due to software version limitations, I was using version 0.1.1 of the code and configuration files. I will update the software and check the results again.
